include/quicly.h

```diff
@@ -331,6 +331,7 @@ struct st_quicly_conn_streamgroup_state_t {
  * the same order for quicly_stats_t and `struct st_quicly_conn_public_t::stats`.
  */
 #define QUICLY_STATS_PREBUILT_FIELDS                                                                                               \
+    uint8_t stats_version;                                                                                                         \
```

Comment:
How about declaring this variable as the first element of `quicly_stats_t`? I do not think there is a reason to "prebuild" this value.

Reply:
I did that to avoid having it in be declared again in `struct _st_quicly_conn_public_t`... but changed it now.